### PR TITLE
feat(components): [button] update text button

### PR DESCRIPTION
**Pull request by jw-foss**
- Update text button style.
- Update API for text button.
- Update documentation for text button.

Screenshot

CleanShot 2022-05-05 at 19 17 43@2x

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by sxzz** [comment]
Is this a breaking change?

**Comment by JeremyWuuuuu** [comment]
> Is this a breaking change?

Yes, it is, in both API and how it looks.

File: docs/.vitepress/vitepress/components/vp-subnav.vue [hunk 1]
```diff
@@ -12,13 +12,13 @@ const { shouldShow, scrollToTop } = useBackTop()
...
+      <ElLink
+        :class="{ 'go-back-top': true, show: shouldShow }"
+        :underline="false"
         class="height-5"
         size="small"
-        :class="{ 'go-back-top': true, show: shouldShow }"
         @click.prevent.stop="scrollToTop"
-        >{{ 'Back to top' }}</ElButton
+        >{{ 'Back to top' }}</ElLink
       >
```
Comment (sxzz) [inline docs/.vitepress/vitepress/components/vp-subnav.vue:22]:
```suggestion
        >
          {{ 'Back to top' }}
        </ElLink>
```

File: docs/.vitepress/vitepress/components/vp-subnav.vue [hunk 2]
```diff
@@ -12,13 +12,13 @@ const { shouldShow, scrollToTop } = useBackTop()
   <div class="sub-nav py-3 flex items-center">
     <ToggleSidebarBtn v-if="hasSidebar" @click="$emit('open-menu')" />
     <Transition name="shifting">
-      <ElButton
-        type="text"
+      <ElLink
+        :class="{ 'go-back-top': true, show: shouldShow }"
```
Comment (sxzz) [inline docs/.vitepress/vitepress/components/vp-subnav.vue:16]:
```suggestion
        :class="['go-back-top', { show: shouldShow }]"
```

**Review by sxzz — APPROVED** [review]

File: docs/.vitepress/vitepress/components/vp-subnav.vue (hunk 1 above)
Comment (JeremyWuuuuu) [reply docs/.vitepress/vitepress/components/vp-subnav.vue:22]: The linter did not say anything about this, I guess we should do something about the linter/formatter?

Comment (sxzz) [reply docs/.vitepress/vitepress/components/vp-subnav.vue:22]: Hmm we can consider it later.

**Comment by sxzz** [comment]
@JeremyWuuuuu 
Please fix https://github.com/element-plus/element-plus/blob/a4aad5a9c0a44578e8062a8acffe68d45e009654/packages/components/popconfirm/src/popconfirm.ts#L25

**Comment by yoyo837** [comment]
站点文档还是旧的。

**Comment by ylater** [comment]
有点不理解，改这个干嘛。升级后要改一堆样式。

**Pull request merged**